keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocCompatibilityTests.kt

```diff
@@ -33,90 +29,55 @@ import strikt.api.expectThat
     "keel.plugins.ec2.enabled=true",
     "keel.plugins.titus.enabled=true"
   ],
-  webEnvironment = MOCK
+  webEnvironment = NONE
 )
 @AutoConfigureMockMvc
 @EnableAutoConfiguration(exclude = [TaskSchedulingAutoConfiguration::class])
-class ApiDocCompatibilityTests : JUnit5Minutests {
+class ApiDocCompatibilityTests {
```

Comment:
This test is pretty valuable as it ensures the schema we generate will validate a delivery config. We can extend this with more edge cases ideally.

Reply:
Is my understanding correct that a `SubmittedDeliveryConfig` successfully parsed with Jackson will have met all the rules in the schema (assuming we've configured Jackson to match the schema)? If not, it would be nice to add a filter that validates the JSON against the schema before it reaches `DeliveryConfigController`.

Reply:
Well, the schema is generated from the classes and parsing rules, but no we're not explicitly validating although we certainly could. Maybe a future change?

Reply:
Sure, future PR sounds good. 👍